worlds/tits_the_3rd/client/memory_io.py

```diff
@@ -256,6 +270,6 @@ def read_flag(self, flag_number: int):
         """
         flag_byte_offset = self.OFFSET_FLAG_0 + (flag_number // 8)
         flag_bit = flag_number % 8
-        data = self._read_byte(flag_byte_offset)
+        data = self._read_bytes(flag_byte_offset)
```

Comment:
Super nit but being able to call a function called "read_bytes" without specifying the number of bytes is a little confusing (unless you go and view the function def to see the default of length=1).

Reply:
yeah agree it is confusing

- [ ] Don't default to 1